### PR TITLE
client, roachpb: eliminate log spam when loadgens are killed

### PR DESCRIPTION
**Pull request by andreimatei**
When the TPCC loadgen is CTRL-C'ed, the logs are spammed with:
I180906 22:02:41.239771 27177127 internal/client/txn.go:625  [n1] async rollback failed: TransactionStatusError: already committed (REASON_UNKNOWN): "sql txn" id=57c518aa key=/Table/61/1/1224/0 rw=true pri=0.03781170 iso=SERIALIZABLE stat=COMMITTED epo=0 ts=1536271361.068457287,0 orig=1536271361.068457287,0 max=1536271361.073978228,0 wto=false rop=false seq=11 int=5

The "async rollback" part refers to the rollback being done with a
canceled ctx (presumably a dropped connection's ctx). I believe the
error happens because there's a commit in flight when the ctx is
canceled. This patch lowers the message's level for this case.

Release note: None

### CODE REVIEW

**Comment by andreimatei** [comment]
cc @jordanlewis

File: pkg/roachpb/errors.go [hunk 1]
```diff
@@ -410,6 +410,15 @@ func NewTransactionNotFoundStatusError() *TransactionStatusError {
 	}
 }
 
+// NewTransactionCommittedStatusError initializes a new TransactionStatusError
+// with a REASON_TXN_COMMITTED.
+func NewTransactionCommittedStatusError() *TransactionStatusError {
+	return &TransactionStatusError{
+		Msg:    "already committed",
```
Comment (jordanlewis) [inline pkg/roachpb/errors.go]: this Msg seems redundant with the Reason - what if you just left it out?

**Review by jordanlewis — APPROVED** [review]
LGTM - thanks!

**Review by andreimatei — COMMENTED** [review]
bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/29817#-:-LM3BB992LZdE_G9VzKL:b-ydi9ay)* status: :shipit: complete! 0 of 0 LGTMs obtained

___
*[pkg/roachpb/errors.go, line 417 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/29817#-LLp5ItQ8IINHy03IRGk-r1-417:-LM3B1WMDvH14sB2CnJ-:b2qrw89) ([raw file](https://github.com/cockroachdb/cockroach/blob/c32a3790c283ec8401d0a776ba0d141cc3d56dab/pkg/roachpb/errors.go#L417)):*
<details><summary><i>Previously, jordanlewis (Jordan Lewis) wrote…</i></summary><blockquote>

this Msg seems redundant with the Reason - what if you just left it out?
</blockquote></details>

meh, I dunno. A bunch of callers still use the legacy catchall reason with a message. I'll leave it.

**Review by andreimatei — COMMENTED** [review]
bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/29817#-:-LM3ifBU1-FnNbISfCfo:b-ydi9ay)* status: :shipit: complete! 0 of 0 LGTMs obtained

**Pull request merged**